src/Client/Shared/Components/FileBrowser/FileBrowser.razor.cs

```diff
@@ -1215,6 +1215,7 @@ private PinOptionResult GetPinOptionResult(List<FsArtifact> artifacts)
     }
 
     private readonly SemaphoreSlim _semaphoreArtifactChanged = new(1);
+    private ArtifactExplorer? _artifactExplorerRef;
```

Comment:
Please add this property next other properties in top of the document.

Reply:
done🌹😊